probcal/data_modules/__init__.py

```diff
@@ -1 +1,4 @@
 from .tabular_data_module import TabularDataModule
+from .coco_people_datamodule import COCOPeopleDataModule
+
+global_data_dir = "data"
```

Comment:
I'm wary of defining anything in the `__init__.py`. Could we move this to a `common/constants.py` file or something?

Reply:
Yeah, no problem. I'll move it